src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -323,6 +393,33 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
             }
         }
 
+        private void SetIndexForBackup(TransactionOperationContext toc, BlittableJsonReaderObject bjro, long index, string type)
+        {
+            if (index < 0)
+                return;
+
+            if ((bjro.TryGet(nameof(CompareExchangeCommandBase.DatabaseName), out string databaseName) == false ||
+                 string.IsNullOrEmpty(databaseName)) && type.Equals(nameof(AddOrUpdateCompareExchangeCommand)))
+                throw new RachisApplyException("Update database command must contain a DatabaseName property");
```

Comment:
Add the type of the command here as well, its index, etc. 
We want to give as much detail as possible.

Reply:
done